src/Features/LanguageServer/Protocol/Handler/SemanticTokens/SemanticTokensHelpers.cs

```diff
@@ -38,6 +38,19 @@ internal class SemanticTokensHelpers
             ClassificationTypeNames.ExtensionMethodName,
             ClassificationTypeNames.FieldName,
             ClassificationTypeNames.InterfaceName,
+
+            ClassificationTypeNames.JsonArray,
+            ClassificationTypeNames.JsonComment,
+            ClassificationTypeNames.JsonConstructorName,
+            ClassificationTypeNames.JsonKeyword,
+            ClassificationTypeNames.JsonNumber,
+            ClassificationTypeNames.JsonObject,
+            ClassificationTypeNames.JsonOperator,
+            ClassificationTypeNames.JsonPropertyName,
+            ClassificationTypeNames.JsonPunctuation,
+            ClassificationTypeNames.JsonString,
+            ClassificationTypeNames.JsonText,
```

Comment:
Instead of needing to do this manually, can we use reflection to grab all the data we need out of classificationtypenames?

Reply:
I think reflection would be cleaner but would complicate things. Razor expects an exact ordering from Roslyn in terms of classifications, so anytime Roslyn updates classifications we need to manually update Razor as well. I think we could eventually do this if we expose RoslynCustomTokenTypes via external access, but I think this work would best be done in a follow-up PR.

Reply:
Filed https://github.com/dotnet/roslyn/issues/61237